Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -86,14 +92,16 @@ internal struct ProtobufCodeGenParser {
 }
 
 extension ProtobufCodeGenParser {
-  fileprivate var codeDependencies: [Dependency] {
+  fileprivate func codeDependencies(
```

Comment:
nit: is are the newlines in the signature intentional/needed?

Reply:
No, I prefer a single line where possible. It was probably just an artefact of how I edited the existing code.